module-new/src/components/checkbox/checkbox.theme.css

```diff
@@ -0,0 +1,71 @@
+button {
+  all: unset;
+}
```

Comment:
This will unset the properties of all buttons globally, regardless of this component. This should be removed and any unsets handled within a more specific selector.